_episodes/08-gitlab-ci.md

```diff
@@ -213,12 +135,17 @@ docker pull <username>/<image name>:<tag>
 ~~~
 {: .source}
 
+> ## Tag your docker image
+> Notice that the command above had a ``<tag>`` specified. A tag uniquely identifies a docker image. When puched to
```

Comment:
This looks incomplete. A missing commit perhaps?

Reply:
Yeah this is finishing on monday evening and starting again on tuesday morning :D